keyboards/rarepotato8de/3x3macropad/readme.md

```diff
@@ -0,0 +1,28 @@
+# 3x3macropad
+
+![3x3macropad](https://i.imgur.com/OTb3CUq.png)
```

Comment:
The image here doesnt match up with the layout macro configured and default keymap